open_event/api/formats.py

```diff
@@ -38,6 +39,13 @@ def get(self, event_id, format_id):
         """Fetch a format given its id"""
         return DAO.get(event_id, format_id)
 
+    @requires_auth
+    @api.doc('delete_format')
+    @api.marshal_with(FORMAT)
+    def delete(self, event_id, format_id):
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://www.codacy.com/app/triplez/open-event-orga-server/file/2716743252/issues/source?bid=3233864&fileBranchId=3345807#l45)